src/ibex_bluesky_core/callbacks/document_logger.py

```diff
@@ -0,0 +1,40 @@
+"""Logs all documents that the BlueSky run engine creates via a callback."""
+
+import json
+import os
+from pathlib import Path
+
+log_location = Path(os.path.join("C:\\", "instrument", "var", "logs", "bluesky", "raw_documents"))
```

Comment:
Since we're using `pathlib` anyway...

```suggestion
log_location = Path("C:\\") / "instrument" / "var" / "logs" / "bluesky" / "raw_documents"
```